lxd/storage.go

```diff
@@ -69,7 +69,7 @@ func storagePoolVolumeAttachPrepare(s *state.State, poolName string, volumeName
 	// Get the on-disk idmap for the volume
 	var lastIdmap *idmap.IdmapSet
 	if poolVolumePut.Config["volatile.idmap.last"] != "" {
-		lastIdmap, err = idmapsetFromString(poolVolumePut.Config["volatile.idmap.last"])
+		lastIdmap, err = storageDrivers.IDMapsetFromString(poolVolumePut.Config["volatile.idmap.last"])
```

Comment:
Would the serialization/desarialization better be done inside of `shared/idmap`?

Reply:
@stgraber yep that makes more sense will move.

Reply:
Have moved to `idmap.JSONUnmarshal()`